sklearn/metrics/_pairwise_distances_reduction/_argkmin.pyx.tp

```diff
@@ -145,8 +145,9 @@ cdef class ArgKmin{{name_suffix}}(BaseDistancesReduction{{name_suffix}}):
                     val=self.datasets_pair.surrogate_dist(
                         X_start,
                         Y_start,
-                        X_start + i,
-                        Y_start + j,
+                        i,
+                        j,
+                        n_samples_Y,
                         thread_num,
```

Comment:
Is it possible to use keyword arguments, here?

This comment also applies to other places calling `DatasetsPair.surrogate_dist`.